filestore/s3_test.go

```diff
@@ -83,7 +83,7 @@ func (t *TestSuite) TestConstructWithGoodBucketNames() {
 	ls := b.String()
 	t.Equal(63, len(ls), "incorrect string length")
 	for _, bucket := range []string{"foo", ls} {
-		fstore, err := NewS3FileStore(cli, min, bucket)
+		fstore, err := NewS3FileStore(cli, min, bucket, false)
```

Comment:
I would disable ssl for one of the happy path tests so the code is exercised, if not tested completely, and add a note to the top of the test file to that effect.